scripts/pages/end-of-run/end-of-run.ts

```diff
@@ -522,7 +522,7 @@ class EndOfRunHandler {
 		this.panels.cp.SetDialogVariable('selected_zone', $.Localize(split.name));
 
 		// Don't scroll for the overall split
-		if (split.name !== 'Run_Comparison_Split_Overall') {
+		if (split.name !== 'Overall') {
 			this.panels.splits.FindChildTraverse(`Split${split.name}`).ScrollParentToMakePanelFit(3, false);
 		}
```

Comment:
This is a localization token; changing this will break i18n. Why change it anyway?